python/cudf/cudf/pandas/scripts/conftest-patch.py

```diff
@@ -36,4 +37,78 @@ def patch_testing_functions():
     pytest.raises = replace_kwargs({"match": None})(pytest.raises)
 
 
+# Dictionary to store function call counts
+function_call_counts = {}  # type: ignore
```

Comment:
Since I think we're running the pandas tests with xdist and this is global, is there a risk that one worker clears these counts too early?